src/mlpack/methods/isomap/k_nearest.hpp

```diff
@@ -0,0 +1,115 @@
+/**
+ * @file k_nearest.hpp
+ * @author Rishabh Ranjan
+ * 
+ * This file defines the class K_Nearest which constructs the neighbourhood
+ * graph for Isomap, according to the number of neighbours provided.
+ * 
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_ISOMAP_K_NEAREST
+#define MLPACK_METHODS_ISOMAP_K_NEAREST
+
+#include <mlpack/prereqs.hpp>
+#include <queue>
+
+namespace mlpack {
+namespace isomap {
+/**
+ * This class implements the K-Nearest Neighbours function to construct
+ * a neighbourhood graph for the given distance matrix.
+*/
```

Comment:
This comment is spaced strangely and away from the class; can you fix that please?

Reply:
Yes surely. Next commit will resolve this.